apps/admin-ui/src/clients/scopes/ClientScopes.tsx

```diff
@@ -78,7 +78,9 @@ export const ClientScopes = ({
   const [addDialogOpen, setAddDialogOpen] = useState(false);
 
   const [rest, setRest] = useState<ClientScopeRepresentation[]>();
-  const [selectedRows, setSelectedRows] = useState<Row[]>([]);
+  const [selectedRows, setSelectedRowState] = useState<Row[]>([]);
+  const setSelectedRows = (rows: Row[]) =>
+    setSelectedRowState([...rows.filter((r) => r.id !== DEDICATED_ROW)]);
```

Comment:
Using `.filter()` already gives you a new array, so it's not required to spread the result.
```suggestion
    setSelectedRowState(rows.filter(({ id }) => id !== DEDICATED_ROW));
```